third_party/move/move-compiler-v2/src/file_format_generator/function_generator.rs

```diff
@@ -943,8 +943,7 @@ impl<'a> FunctionGenerator<'a> {
                     // Copy the temporary if it is copyable and still used after this code point, or
                     // if it appears again in temps_to_push.
                     if fun_ctx.is_copyable(*temp)
```

Comment:
Is the `is_copyable` check here required? If it is alive after, does it not have to be copyable (if not, shouldn't previous compiler checks have caught it)?

Reply:
Good observation. Instead of making this a branch condition, I now generate an internal (bug) error if the value needs copy but is not copyable.